test-utils/src/main/java/com/ibm/ws/lars/testutils/clients/LooseFileWriteableClient.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+* Copyright (c) 2017 IBM Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+package com.ibm.ws.lars.testutils.clients;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+
+import com.ibm.ws.repository.transport.client.DirectoryUtils;
+import com.ibm.ws.repository.transport.client.LooseFileClient;
+import com.ibm.ws.repository.transport.exceptions.BadVersionException;
+import com.ibm.ws.repository.transport.exceptions.RequestFailureException;
+import com.ibm.ws.repository.transport.model.Asset;
+import com.ibm.ws.repository.transport.model.Attachment;
+import com.ibm.ws.repository.transport.model.AttachmentSummary;
+
+/**
+ *
+ */
+public class LooseFileWriteableClient extends AbstractFileWriteableClient {
+
+    Collection<String> _assets;
+
+    public LooseFileWriteableClient(Collection<String> assets) {
+        _assets = assets;
+        _readClient = new LooseFileClient(assets);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Attachment addAttachment(String assetId, AttachmentSummary attSummary) throws IOException, BadVersionException, RequestFailureException, SecurityException {
+        // There should be no attachments
+        return null;
```

Comment:
Worth throwing an exception here?

Reply:
Yep, it should be throwing an exception indeed. Updated.